src/app/modules/form/admin-form/__tests__/admin-form.service.spec.ts

```diff
@@ -2628,4 +2630,124 @@ describe('admin-form.service', () => {
       expect(twilioCacheSpy).toHaveBeenCalledWith(MSG_SRVC_NAME)
     })
   })
+
+  describe('updatePayments', () => {
+    // Arrange
+    const mockFormId = new ObjectId().toString()
+    const updatedPaymentSettings = {
+      enabled: true,
+      target_account_id: 'someId',
+      publishable_key: 'somekey',
+      amount_cents: 5000,
+      description: 'some description',
+    } as PaymentsUpdateDto
+
+    const mockUpdatedForm = {
+      _id: mockFormId,
+      payments: updatedPaymentSettings,
+    }
+
+    it('should return InvalidPaymentAmountError if payment amount exceeds maxPaymentAmountCents', async () => {
+      // Arrange
+
+      const updatedPaymentSettingsExceeded = {
+        enabled: true,
+        target_account_id: 'someId',
+        publishable_key: 'somekey',
+        amount_cents: 500000,
+        description: 'some description',
+      } as PaymentsUpdateDto
+
+      // Act
+      const actualResult = await AdminFormService.updatePayments(
+        mockFormId,
+        updatedPaymentSettingsExceeded,
+      )
+
+      // Assert
+      expect(actualResult.isErr()).toEqual(true)
+      expect(actualResult._unsafeUnwrapErr()).toBeInstanceOf(
+        InvalidPaymentAmountError,
+      )
+    })
+
+    it('should return InvalidPaymentAmountError if payment amount is below minPaymentAmountCents', async () => {
+      // Arrange
+
+      const updatedPaymentSettingsBelow = {
+        enabled: true,
+        target_account_id: 'someId',
+        publishable_key: 'somekey',
+        amount_cents: 49,
+        description: 'some description',
+      } as PaymentsUpdateDto
```

Comment:
Same comment as above, making use of spread (`...`) might be better:
```suggestion
      const updatedPaymentSettingsBelow = {
        ...updatedPaymentSettings,
        amount_cents: 49,
      } as PaymentsUpdateDto
```